### PR TITLE
release: propose release candiate 1

### PR DESCRIPTION
**Pull request by Fraser-Isbester**
Proposes a first release with a minimum feature set.

### CODE REVIEW

File: pulse/loaders/slack.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        logger.info("Ignoring event of type %s", event.type)
+        return
+
+    text = event.text
+    meta = {k: v for k, v in event.__dict__.items() if k != "text"}
+    meta["source"] = "slack"
+
+    logger.debug("Adding message to vectorstore: %s, {%s}", text, meta)
+
+    id = await redis.aadd_texts(texts=[text], metadata=meta)
```
Comment (allisonbraithw) [inline pulse/loaders/slack.py:27]: typo?

File: pulse/services/slack.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    content = history["messages"][0]["text"] if history.get("messages") else None
+    return content
+
+
+def post_message(client, channel: str, text: str):
+    """Posts a message to a Slack channel."""
+
+    logging.debug("Posting message to channel %s: %s", channel, text)
+    client.chat_postMessage(channel=channel, text=text)
```
Comment (allisonbraithw) [inline pulse/services/slack.py:72]: i think replying in a thread is cleaner

File: pulse/routers/slack.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    ## Event Processing Below ##
+
+    # TODO: push this to a processing queue
+    await slack_event_loader(request.event)
+
+    chat_completion_retriever = get_retriever()
+    match request.event:
+        # TODO: push this to a processing queue
+        case event if event.type == EventTypes.REACTION_ADDED.value and event.reaction == "sos":
```
Comment (allisonbraithw) [inline pulse/routers/slack.py:53]: no :eyes:????

File: pulse/services/slack.py (hunk 2 above)
Comment (Fraser-Isbester) [reply pulse/services/slack.py:72]: Oh, how do I do that?

File: pulse/loaders/slack.py (hunk 1 above)
Comment (Fraser-Isbester) [reply pulse/loaders/slack.py:27]: nope, aadd_texts == `async add_tests`

File: pulse/routers/slack.py (hunk 3 above)
Comment (Fraser-Isbester) [reply pulse/routers/slack.py:53]: haha yeah lemme go back to eyes, was playing with how that works

File: pulse/services/slack.py (hunk 2 above)
Comment (allisonbraithw) [reply pulse/services/slack.py:72]: `thread_ts=<ts>`

File: pulse/loaders/slack.py (hunk 1 above)
Comment (allisonbraithw) [reply pulse/loaders/slack.py:27]: ty for the learning!

**Review by allisonbraithw — DISMISSED** [review]
yay!

File: pulse/routers/slack.py (hunk 3 above)
Comment (allisonbraithw) [reply pulse/routers/slack.py:53]: sos is fine, I just did eyes bc it was in the default set so less clicking lol

File: pulse/services/slack.py (hunk 2 above)
Comment (Fraser-Isbester) [reply pulse/services/slack.py]: Screenshot 2023-11-28 at 7 50 37 PM
tada!

**Review by allisonbraithw — APPROVED** [review]

**Pull request merged**